core/metadata.js

```diff
@@ -253,7 +253,7 @@ function extractRevNumber (doc /*: Metadata|{_rev: string} */) {
 function isUpToDate (side /*: SideName */, doc /*: Metadata */) {
   let currentRev = doc.sides[side] || 0
   let lastRev = extractRevNumber(doc)
-  return currentRev === lastRev
+  return currentRev >= lastRev
```

Comment:
The commit says the opposite of the code: _up to date when last rev >= current one_.

I think the code (`currentRev >= lastRev`) is right, and you should fix the commit message.

Reply:
Please note the goal is not necessarily to merge this change, but just to see whether it breaks something (as part of discussion on #1250 and #1273)